package.json

```diff
@@ -23,9 +23,29 @@
     "body-parser": "^1.19.0",
     "chalk": "^4.1.0",
     "chalk-animation": "^1.6.0",
+    "axios": "^0.19.2",
```

Comment:
Hey, you added all these packages to package.json AND NEVER ADDED YOUR NAME AS AUTHOR.

Reply:
@cofuente noted!